<launch>
  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" />
  <node pkg="web_teleop" type="head_node.py" name="web_teleop_head" />
  <node pkg="web_teleop" type="eyelids_node.py" name="web_teleop_eyelids" />
  <node pkg="web_teleop" type="pose_node.py" name="web_teleop_pose" />
  <node pkg="web_video_server" type="web_video_server" name="web_video_server">
    <param name="port" value="8000" />
  </node>
  <node pkg="tf2_web_republisher" type="tf2_web_republisher" name="tf2_web_republisher" />
  <node pkg="interactive_marker_proxy" type="proxy" name="interactive_marker_proxy">
    <param name="topic_ns" value="/basic_controls" />
    <param name="target_frame" value="/odom" />
  </node>
</launch>
